.github/workflows/changelog-checker.yml

```diff
@@ -76,6 +76,12 @@ jobs:
           elif grep -q ':fix$' "$changelog_files"; then
             echo "Found invalid type (fix) in changelog - did you mean bug?"
             exit 1
+          elif grep -q ':feature$' "$changelog_files"; then
+            if ! grep -q '^\*\*' "$changelog_files"; then
```

Comment:
`^` is the 'start of line' character for grep, so if a line doesn't start with `**`, this condition will be satisfied